lib/routes/home/widgets/payments_list/widgets/payment_details_sheet/widgets/payment_details_sheet_bip_353_address.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:l_breez/widgets/widgets.dart';
+
+class PaymentDetailsSheetBip353Address extends StatelessWidget {
+  final String bip353Address;
+
+  const PaymentDetailsSheetBip353Address({required this.bip353Address, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData themeData = Theme.of(context);
+
+    return ShareablePaymentRow(
+      tilePadding: EdgeInsets.zero,
+      dividerColor: Colors.transparent,
+      title: 'BIP 353 Address:',
```

Comment:
nit: I think BIP 353 is too technical. "Lightning Address" should suffice, same as on the Send input

Reply:
Done. 
- [Use "Lightning Address:" as title for BIP 353 Addresses on payment details](https://github.com/breez/misty-breez/pull/356/commits/124409825af4e0ec940dca9c1d624d8d102f9add)